words/scales.txt

```diff
@@ -179,3 +179,18 @@ beaver
 hops
 salak
 mercat
+great
```

Comment:
I don't think penguins have scales?

Reply:
![king-penguin-feet-antarctica-philippe-tulula-and-julie-alexander](https://user-images.githubusercontent.com/46385858/202863917-b8408234-e9e7-4e11-80be-ed98e2beccf7.jpg)

Reply:
> Do Penguins Have Scales?

Every part of a penguin’s body, from its head to its feet, is coated in scales and feathers.

Reply:
🤯

Reply:
I'm also very happy that this means that `great-great` is now a valid tailnet name 🤣

Reply:
great-gentoo, linux fans are gonna go NUTS